portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetFragment.testcase

```diff
@@ -36,881 +36,6 @@ definition {
-				key_rowPosition = 2,
-				locator1 = "DataSet#VIEW_LABEL_POSITION");
-		}
-	}
-
-	@description = "LPS-185993. Confirm that you can configure the order by date of a fragment column"
-	@priority = 4
-	test ConfirmNewSortingByDate {
-		task ("Given a new Object entity") {
-			ObjectAdmin.addObjectViaAPI(
-				labelName = "Date Object",
-				objectName = "DateObject",
-				pluralLabelName = "Date Objects");
-
-			ObjectAdmin.addObjectFieldViaAPI(
-				fieldBusinessType = "Date",
-				fieldLabelName = "Date Field",
-				fieldName = "dateField",
-				fieldType = "Date",
-				isRequired = "false",
-				objectName = "DateObject");
-
-			ObjectAdmin.publishObjectViaAPI(objectName = "DateObject");
-		}
-
-		task ("And new entries in the new Object entity") {
-			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
-				fieldName = "dateField",
-				objectName = "DateObject",
-				value = "2022-02-02");
-
-			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
-				fieldName = "dateField",
-				objectName = "DateObject",
-				value = "2022-01-01");
-		}
-
-		task ("And creating a Data Set entry, view and fields") {
-			DataSetAdmin.createAndConfigureDataSetEntry(
-				dataSetName = "Custom Data Set Entry",
-				dataSetViewDescription = "View Description",
-				dataSetViewName = "Dates view",
-				key_fieldList = "dateField,id,dateCreated",
-				restApplication = "/c/dateobjects",
-				restEndpoint = "/",
-				restSchema = "DateObject");
-		}
-
-		task ("And Add a default sorting") {
-			DataSetAdmin.goToTab(tabName = "Sorting");
-
-			DataSetAdmin.createASort(
-				key_field = "dateField",
-				key_sorting = "Descending");
-		}
-
-		task ("And adds the data set Fragment portlet in the page created") {
-			PagesAdmin.openPagesAdmin(siteURLKey = "test-site-name");
-
-			ContentPagesNavigator.openEditContentPage(
-				pageName = "Data Set Test Page",
-				siteName = "test-site-name");
-
-			PageEditor.addFragment(
-				collectionName = "Content Display",
-				fragmentName = "Data Set");
-		}
-
-		task ("And adds the Dates view to the data set Fragment portlet") {
-			DataSetFragment.addDataSetView(dataSetViewName = "Dates view");
-		}
-
-		task ("And publishes the page") {
-			var portalURL = PropsUtil.get("portal.url");
-
-			SelectFrameTop();
-
-			Button.clickPublish();
-
-			PagesAdmin.openPagesAdmin(siteURLKey = "test-site-name");
-
-			Navigator.openSpecificURL(url = "${portalURL}/web/test-site-name/data-set-test-page");
-		}
-
-		task ("Then Confirm that the table is ordered by the dateField in descending order") {
-			AssertElementPresent(
-				key_labelName = "2022-02-02T00:00:00.000Z",
-				key_rowPosition = 1,
-				locator1 = "DataSet#VIEW_LABEL_POSITION");
-
-			AssertElementPresent(
-				key_labelName = "2022-01-01T00:00:00.000Z",
-				key_rowPosition = 2,
-				locator1 = "DataSet#VIEW_LABEL_POSITION");
-		}
-	}
-
 	@description = "LPS-172403. Confirm that the select view button changes to the name of the selected view"
 	@priority = 4
```

Comment:
Why did this one survive?

Reply:
This tests will be removed / migrated with the fix for https://liferay.atlassian.net/browse/LPD-26897